scanner/keppel/main.go

```diff
@@ -129,11 +129,34 @@ func HandleRepository(fqdn string, account models.Account, repository models.Rep
 			}).Error("Component not found")
 			return
 		}
+		if manifest.VulnerabilityStatus == "Unsupported" {
+			log.WithFields(log.Fields{
+				"account:":   account.Name,
+				"repository": repository.Name,
+			}).Warn("Manifest has UNSUPPORTED type: " + manifest.MediaType)
+			continue
+		}
+		if manifest.VulnerabilityStatus == "Clean" {
+			log.WithFields(log.Fields{
+				"account:":   account.Name,
+				"repository": repository.Name,
+			}).Info("Manifest has no Vulnerabilities")
+			continue
+		}
 		HandleManifest(account, repository, manifest, component, keppelScanner, keppelProcessor)
 	}
 }
 
 func HandleManifest(account models.Account, repository models.Repository, manifest models.Manifest, component *client.Component, keppelScanner *scanner.Scanner, keppelProcessor *processor.Processor) {
+	childManifests, err := keppelScanner.ListManifestsOfManifest(account.Name, repository.Name, manifest.Digest)
```

Comment:
I don't think "ListManifestsOfManifests" is a proper name for a method. Why don't you call it "ListChildManifests" whereas the 1st parameter is a manifest (followed by account.Name, repository.Name etc.)?

Reply:
I agree. This is about processing `Multi Arch` Images in Keppel. HandleMultiArchImage or something similar would be clearer IMO

Reply:
Well mutli arch are just the most prominent. I'd prefer "ListChildManifsts" then

Reply:
done